### PR TITLE
feat(surveys): Replace minimize/maximize functionality with close

### PR DESCRIPTION
**Pull request by LZoog**
## Because

* Legal requested we change the functionality to close.

## This commit

* Changes the minimize/maximize 'survey-control' button with a 'survey-close' button, including a swap from chevron to close icon.
* Fixes Storybook styles.

## Issue that this pull request solves

Closes: #5842

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [x] I have added necessary documentation (if appropriate).

## Other information

#5870 is a spin-off ticket of an importing issue I ran into while working on resolving this. I added some comments referencing that ticket and removed a related TODO comment.

### CODE REVIEW

File: packages/fxa-content-server/app/scripts/views/survey.jsx [hunk 1]
```diff
@@ -34,7 +38,7 @@ const SurveyView = (props) => {
     return () => window.removeEventListener('message', handleIframeTask);
   }, [setShowSurvey, setSurveyComplete, setTimeout]);
 
-  return <>{showSurvey && <Survey {...{ surveyComplete, surveyURL }} />}</>;
+  return <>{showSurvey && <Survey {...{ surveyComplete, surveyURL, onSurveyClose }} />}</>;
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/scripts/views/survey.jsx:41]: I see this isn't part of your changes, but... is the fragment necessary?

File: packages/fxa-content-server/app/styles/modules/_survey.scss [hunk 2]
```diff
@@ -1,3 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// If you make changes here, be sure to update
+// `fxa-react/components/Survey/index.scss` for Storybook. See FXA-2237
```
Comment (jodyheavener) [inline packages/fxa-content-server/app/styles/modules/_survey.scss:6]: thank you for adding this

File: packages/fxa-content-server/app/scripts/views/survey.jsx (hunk 1 above)
Comment (jodyheavener) [reply packages/fxa-content-server/app/scripts/views/survey.jsx:41]: I see this a few times below as well, I think you can get away with just `return (condition && <Component />)`, no?

Comment (LZoog) [reply packages/fxa-content-server/app/scripts/views/survey.jsx:41]: Looks like that's the equivalent of `return false` when `showSurvey` is `false` - no complains in the JSX file, but Storybook has to have a React element returned. I can make that change here, will wait on other comments. 👍

Comment (jodyheavener) [reply packages/fxa-content-server/app/scripts/views/survey.jsx:41]: Ah okay as long as it's intentional! Carry on.

**Pull request merged**